spring-integration-core/src/test/java/org/springframework/integration/config/ChainParserTests-context.xml

```diff
@@ -125,7 +125,7 @@
 
 	<chain id="subComponentsIdSupport1" input-channel="subComponentsIdSupport1Channel">
 		<splitter id="splitterWithinChain"/>
-		<resequencer id="resequencerWithinChain"/>
+		<resequencer id="resequencerWithinChain" correlation-strategy="errorChannel" release-strategy="errorChannel"/>
```

Comment:
I think we need to revise the parser and avoid such a change in the XML configs.
See Java DSL for example:
```
	/**
	 * Populate the
	 * {@link org.springframework.integration.aggregator.ResequencingMessageHandler} with
	 * default options.
	 * @return the current {@link BaseIntegrationFlowDefinition}.
	 */
	public B resequence() {
```
so, having `<resequencer/>` in the XML is totally fine.
If that is otherwise, the respective XML parser has to be fixed.